gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -631,7 +630,7 @@ public SeekableByteChannel open(
    * behavior.
    */
   @Override
-  public SeekableByteChannel open(
+  public GoogleCloudStorageReadableByteChannel open(
```

Comment:
What is the problem if we still return SeekableBytChannel?

Reply:
then returned channel will not have new api we want to add i.e. readVectored